frontend/server/src/test/java/com/amazonaws/ml/mms/ModelServerTest.java

```diff
@@ -336,6 +339,7 @@ public void initChannel(Channel ch) {
                                     if (configManager.isUseSsl()) {
                                         p.addLast(sslCtx.newHandler(ch.alloc()));
                                     }
+                                    p.addLast(new ReadTimeoutHandler(30));
```

Comment:
Is this for the worker timeout?

Reply:
No, this is unit test client read time out. Nothing to do with worker.